src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
```

Comment:
```suggestion
    RED,
```
прийнято писати значення enum через капс